docs/getting-started/dashboard_tutorial.md

```diff
@@ -0,0 +1,1788 @@
+            rx.table.row(
+                rx.table.cell("Zahra Ambessa"),
+                rx.table.cell("zahra@example.com"),
+                rx.table.cell("Female"),
+            ),
+        ),
+    )
+```
+
+Components in Reflex have `props`, which can be used to customize the component and are passed in as keyword arguments to the component function. 
+
+The `rx.table.root` component has for example the `variant` and `size` props, which customize the table as seen below.
+
+```python eval
+rx.table.root(
+        rx.table.header(
+            rx.table.row(
+                rx.table.column_header_cell("Name"),
+                rx.table.column_header_cell("Email"),
+                rx.table.column_header_cell("Gender"),
+            ),
+        ),
+        rx.table.body(
+            rx.table.row(
+                rx.table.cell("Danilo Sousa"),
+                rx.table.cell("danilo@example.com"),
+                rx.table.cell("Male"),
+            ),
+            rx.table.row(
+                rx.table.cell("Zahra Ambessa"),
+                rx.table.cell("zahra@example.com"),
+                rx.table.cell("Female"),
+            ),
+        ),
+        variant="surface",
+        size="3",
+    )
+```
+
+```python
+def index() -> rx.Component:
+    return rx.table.root(
+        rx.table.header(
+            rx.table.row(
+                rx.table.column_header_cell("Name"),
+                rx.table.column_header_cell("Email"),
+                rx.table.column_header_cell("Gender"),
+            ),
+        ),
+        rx.table.body(
+            rx.table.row(
+                rx.table.cell("Danilo Sousa"),
+                rx.table.cell("danilo@example.com"),
+                rx.table.cell("Male"),
+            ),
+            rx.table.row(
+                rx.table.cell("Zahra Ambessa"),
+                rx.table.cell("zahra@example.com"),
+                rx.table.cell("Female"),
+            ),
+        ),
+        variant="surface",
+        size="3",
+    )
+```
+
+## Showing dynamic data (State)
+
+Up until this point all the data we are showing in the app is static. This is not very useful for a data dashboard. We need to be able to show dynamic data that can be added to and updated.
+
+This is where `State` comes in. `State` is a Python class that stores variables that can change when the app is running, as well as the functions that can change those variables.
+
+To define a state class, subclass `rx.State` and define fields that store the state of your app. The state variables (vars) should have a type annotation, and can be initialized with a default value. Check out the [basics]({docs.getting_started.basics.path}) section for a simple example of how state works.
+
+
+In the example below we define a `State` class called `State` that has a variable called `users` that is a list of lists. Each list in the `users` list represents a user and contains their name, email and gender.
+
+```python
+class State(rx.State):
```

Comment:
I think my general opinion is that introducing state classes with a class called ``State'' is weird. Because then a question arises as to whether that keyword should especially used for naming the State class or not. But I don't know, if this is the standard for the rest of the docs then it's fine.

Reply:
I think this is fairly standard as it's what we do in the introduction and basics